languages/ruby/ruby_on_rails.md

```diff
@@ -106,20 +121,18 @@ extract value objects (and place in app/models)
 extract service objects (and place in app/services)
 extract form objects (and place in app/forms)
 extract query objects (and place in app/models)
-introduce view objects
-extract policy objects
+introduce view objects (helpers, ViewComponents and Decorators are all good directions to think in)
 extract decorators (consider using the draper gem)
+extract policy objects (there are a number of good gems for this)
 
 # Security
 
-As a starting point, please read http://guides.rubyonrails.org/security.html and become familiar with the concepts.  We must adhere to all of the points mentioned there.
-
-Another useful, quick read is http://blog.codeclimate.com/blog/2013/03/27/rails-insecure-defaults/
+As a starting point, please read https://guides.rubyonrails.org/security.html and become familiar with the concepts.  We must adhere to all of the points mentioned there.
```

Comment:
Do we want to mention response to Dependabot alerts in this section? i.e. what is the expected mitigation approach to brakeman and/or dependabot alerts. 

- Low
- Medium
- High
- Critical

Reply:
Or maybe a separate section after **Brakeman** to go over automated code security threat response.

Reply:
I think in many cases it might depend on the organization and the app.  There is a brakeman section below and I think it's OK as a overview.

Reply:
great. lots of these comments are stream-of-consciousness. agreed we don't want this to attempt to be all-encompassing.